parser/src/lexer/tests/random_values.rs

```diff
@@ -0,0 +1,86 @@
+use super::{expect_valid_tokenization, Token};
+
+#[test]
+fn random_values_section() {
+    let source = "random_values";
+    let tokens = vec![Token::RandomValues];
+    expect_valid_tokenization(source, tokens);
+}
+
+#[test]
+fn random_values_empty_list_default_name() {
+    let source = "
+random_values:
+    rand: []";
+
+    let tokens = vec![
+        Token::RandomValues,
+        Token::Colon,
+        Token::Ident("rand".to_string()),
+        Token::Colon,
+        Token::Lsqb,
+        Token::Rsqb,
+    ];
+    expect_valid_tokenization(source, tokens);
+}
+
+#[test]
+fn random_values_fixed_list_default_name() {
+    let source = "
+random_values:
+    rand: [15]";
+
+    let tokens = vec![
+        Token::RandomValues,
+        Token::Colon,
+        Token::Ident("rand".to_string()),
+        Token::Colon,
+        Token::Lsqb,
+        Token::Num("15".to_string()),
+        Token::Rsqb,
+    ];
+    expect_valid_tokenization(source, tokens);
+}
+
+#[test]
+fn random_values_fixed_list_custom_name() {
+    let source = "
+random_values:
+    alphas: [15]";
+
+    let tokens = vec![
+        Token::RandomValues,
+        Token::Colon,
+        Token::Ident("alphas".to_string()),
+        Token::Colon,
+        Token::Lsqb,
+        Token::Num("15".to_string()),
+        Token::Rsqb,
+    ];
+    expect_valid_tokenization(source, tokens);
+}
+
+#[test]
+fn random_values_ident_vector_default_name() {
+    let source = "
```

Comment:
I would rename this test, since the relevant thing that it's testing is the group of values `b` (for the lexer `rand` vs `alphas` above is equivalent)